### PR TITLE
New package: Samsung.PhoneShare version 1.0.4.5

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

### CODE REVIEW

File: manifests/s/Samsung/PhoneShare/1.0.4.5/Samsung.PhoneShare.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.PhoneShare
+PackageVersion: 1.0.4.5
+PackageLocale: en-US
+Publisher: Samsung
+PackageName: PhoneShare
+License: Proprietar
```
Comment (Trenly) [inline manifests/s/Samsung/PhoneShare/1.0.4.5/Samsung.PhoneShare.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
```

File: manifests/s/Samsung/PhoneShare/1.0.4.5/Samsung.PhoneShare.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+- InstallerUrl: https://org.downloadcenter.samsung.com/downloadfile/ContentsFile.aspx?CDSite=US&CttFileID=4590939&CDCttType=SW&ModelType=C&ModelName=XE700T1A-A02US&VPath=SW/201110/20111014145350193/PhoneShare_1.0.4.5.ZIP
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe
+  NestedInstallerFiles:
+  - RelativeFilePath: Inst.exe
+  InstallerSha256: 128A1EA6F578E7587DF382870841E59A57BE407E67081C72ABB8650232238084
+  InstallerSwitches:
+    Silent: /S
+    SilentWithProgress: /passive
```
Comment (Trenly) [inline manifests/s/Samsung/PhoneShare/1.0.4.5/Samsung.PhoneShare.installer.yaml:16]:
```suggestion
  NestedInstallerType: burn
  NestedInstallerFiles:
  - RelativeFilePath: Inst.exe
  InstallerSha256: 128A1EA6F578E7587DF382870841E59A57BE407E67081C72ABB8650232238084
```

File: manifests/s/Samsung/PhoneShare/1.0.4.5/Samsung.PhoneShare.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.PhoneShare
+PackageVersion: 1.0.4.5
+Installers:
+- InstallerUrl: https://org.downloadcenter.samsung.com/downloadfile/ContentsFile.aspx?CDSite=US&CttFileID=4590939&CDCttType=SW&ModelType=C&ModelName=XE700T1A-A02US&VPath=SW/201110/20111014145350193/PhoneShare_1.0.4.5.ZIP
```
Comment (Trenly) [inline manifests/s/Samsung/PhoneShare/1.0.4.5/Samsung.PhoneShare.installer.yaml:7]:
```suggestion
- InstallerUrl: https://downloadcenter.samsung.com/content/SW/201110/20111014145350193/PhoneShare_1.0.4.5.ZIP
```

**Review by Trenly — CHANGES_REQUESTED** [review]
The application doesn't seem to install anything.

Using the MSI or other exe in the folder shows a dependency on .NET 3.5
image

**Comment by upintheairsheep** [comment]
> The application doesn't seem to install anything.
> 
> Using the MSI or other exe in the folder shows a dependency on .NET 3.5 image

This appears to be a dependency

**Comment by upintheairsheep** [comment]
> > The application doesn't seem to install anything.
> > Using the MSI or other exe in the folder shows a dependency on .NET 3.5 image
> 
> This appears to be a dependency

You forgot to tag this.

**Comment by ItzLevvie** [comment]
[Policy] Dependencies

**Comment by ItzLevvie** [comment]
[Policy] Installer-Issue

**Pull request closed without merging**